packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/ports.html

```diff
@@ -309,23 +317,34 @@ <h3>
                             id="internalPort"
                             class="oui-input"
                             data-ng-model="port.tempValue.internalPort"
-                            placeholder="{{PortCtrl.PACK_XDSL.internalPortPlaceHolder}}"
                             maxlength="{{PortCtrl.PACK_XDSL.maxlength}}"
                             min="{{PortCtrl.PACK_XDSL.min}}"
+                            max="{{PortCtrl.PACK_XDSL.max}}"
                             required
-                            max="{{PortCtrl.PACK_XDSL.maxlength}}"
+                            placeholder="{{PortCtrl.PACK_XDSL.internalPortPlaceHolder}}"
                         />
+                    </td>
+                    <td
+                        data-title="{{ 'xdsl_modem_ports_internal_port_end' | translate }}"
+                        class="form-group"
+                        data-ng-class="{ 'has-error': localForm.internalPortEnd.$touched && localForm.internalPortEnd.$invalid }"
+                    >
+                        <label
```

Comment:
Maybe we could use a `oui-field` ?

Reply:
I note this for a next feature, because if I do that, this field is not in the same style like the others. So I'll do it next time for all elements of the table (UXCT-407).